atomix/templates/nodeports.yaml

```diff
@@ -0,0 +1,19 @@
+# Sometimes, it is very hard monitoring or taking the snapshots through the proxy
+{{- if .Values.profiler.enabled }}
```

Comment:
See my comment in ONOS helm chart PR #407. I'd suggest renaming this to
`Values.profiler.nodePort` to avoid confusion.